/**:
  ros__parameters:
    #set Number of cars 
    car_number: 1
  
    #set cars position = [x1, y1, deg1, x2, y2, deg2...]
    #car_pos: [6.0, 0.0, 0.0, -6.0, 0.0, 0.0, 0.0, 4.0, 0.0, 0.0, -4.0, 0.0]
    car_pos: [10.0, 10.0, 0.0]
